mrjob/tools/spark_submit.py

```diff
@@ -26,6 +26,10 @@
    added ``local``, ``spark`` runners, made ``spark`` the default (was
    ``hadoop``)
 
+.. versionchanged:: 0.7.1
```

Comment:
Should it be 0.7.2 or 0.7.2.dev0?

Reply:
Should be v0.7.2.dev0. This is fixed in master